### PR TITLE
Feature/groomer profile page

### PR DESCRIPTION
**Pull request by mikejeter**
## Description ##

This pull request is a new feature added 'Groomer Profile Page' added groomer-profile to router, created Groomer Profile to components/pages, created GroomerProfileContainer.js, index.js, RenderGroomerProfilePage.js created env.sample to fix bug, reinstalled storybook to add UI addons.

## Type ##

- [x] This is a bugfix.
- [x] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [x] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [ ] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [ ] Have you added two reviewers to this pull request?

### CODE REVIEW

**Comment by mikejeter** [comment]
[Create individual Groomer profile page visible to user upon clicking on search results](https://trello.com/c/RM6X5orM/27-create-individual-groomer-profile-page-visible-to-user-upon-clicking-on-search-results)

File: .storybook/main.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 module.exports = {
```
Comment (ShawnBatson) [inline .storybook/main.js:1]: I've never seen Storybook used before.  I'm glad you chose to use it. I will be hopefully using your use of it as a guideline to learn myself. Thank you.

File: src/stories/Button.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
+  const mode = primary
```
Comment (ShawnBatson) [inline src/stories/Button.js:9]: Nice use of Ternary

**Review by ShawnBatson — DISMISSED** [review]
Nicely done. I've never see storybook used before, but your work helped me understand it's use.  Thank you.

**Review by ShawnBatson — APPROVED** [review]

**Review by ShawnBatson — DISMISSED** [review]

**Review by ShawnBatson — APPROVED** [review]

File: src/components/common/ProfileArea.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function ProfileArea(props) {
+  return (
+    <div>
```
Comment (Broast42) [inline src/components/common/ProfileArea.js:6]: This looks like a better way to display the info than using the ant component you were trying to use. Although some of the ant design components might still be able to be used here.

**Review by Broast42 — APPROVED** [review]
Good job. Can't wait to see how it turns out.

**Pull request merged**